src/main/java/core/basesyntax/BuilderTestApp.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
 public class BuilderTestApp {
-  public static void main(String[] args) {
-      // Remove this comment and test your Builder implementation here...
-  }
+
+    public static void main(String[] args) {
+        Plane.PlaneBuilder planeBuilder = new Plane.PlaneBuilder()
+                .setModel("Airbus 383")
+                .setFlyDistance(1500)
+                .setPassengersCapacity(118)
+                .setIsIntercontinental(true)
+                .setHasBusinessClass(false);
+
+        Plane plane = planeBuilder.build();
+
+        plane.toString();
```

Comment:
how can we see the result of this statement?

Reply:
done

Reply:
done